qa/L0_batcher/test.sh

```diff
@@ -694,4 +722,4 @@ else
     echo -e "\n***\n*** Test FAILED\n***"
 fi
 
-exit $RET
+exit $RET
```

Comment:
Add new line at the end of the file.

Reply:
Should have the newline in the latest commit.